### PR TITLE
docs(www): Fix Internal links in Debugging page

### PR DESCRIPTION
**Pull request by Jezfx**
## Description

Fix for this page: 
When you click the first two internal links (Auto-Config, Manual-Config) they don't work. 
https://www.gatsbyjs.org/docs/debugging-the-build-process/#vs-code-debugger-manual-config

Removing the parentheses fixes the internal target. 

## Related Issues
None

### CODE REVIEW

File: docs/docs/debugging-the-build-process.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ Gatsby's `build` and `develop` steps run as a Node.js application which you can
 
 In this guide you will learn how to debug some code using:
 
-- [VS Code debugger](<#vs-code-debugger-(auto-config)>) (Auto-Config)
-- [VS Code debugger](<#vs-code-debugger-(manual-config)>) (Manual-Config)
+- [VS Code debugger](<#vs-code-debugger-auto-config>) (Auto-Config)
```
Comment (muescha) [inline docs/docs/debugging-the-build-process.md]:
```suggestion
- [VS Code debugger (Auto-Config)](<#vs-code-debugger-auto-config>)
```

File: docs/docs/debugging-the-build-process.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ Gatsby's `build` and `develop` steps run as a Node.js application which you can
...
+- [VS Code debugger](<#vs-code-debugger-manual-config>) (Manual-Config)
```
Comment (muescha) [inline docs/docs/debugging-the-build-process.md]:
```suggestion
- [VS Code debugger (Manual-Config)](<#vs-code-debugger-manual-config>)
```

File: docs/docs/debugging-the-build-process.md (hunk 1 above)
Comment (muescha) [reply docs/docs/debugging-the-build-process.md]: it's part of the title - so it should also be in link

Comment (muescha) [reply docs/docs/debugging-the-build-process.md]: to avoid the 2 lines looks like they point to the same anchor

Comment (Jezfx) [reply docs/docs/debugging-the-build-process.md]: @muescha yeah, agree thanks 👌

**Comment by Jezfx** [comment]
@muescha any idea why the linter is failing on this?

File: docs/docs/debugging-the-build-process.md [hunk 3]
```diff
@@ -6,8 +6,8 @@ Gatsby's `build` and `develop` steps run as a Node.js application which you can
 
 In this guide you will learn how to debug some code using:
 
-- [VS Code debugger](<#vs-code-debugger-(auto-config)>) (Auto-Config)
-- [VS Code debugger](<#vs-code-debugger-(manual-config)>) (Manual-Config)
+- [VS Code debugger (Auto-Config)](<#vs-code-debugger-auto-config>)
```
Comment (sidharthachatterjee) [inline docs/docs/debugging-the-build-process.md]:
```suggestion
- [VS Code debugger (Auto-Config)](#vs-code-debugger-auto-config)
```

File: docs/docs/debugging-the-build-process.md [hunk 4]
```diff
@@ -6,8 +6,8 @@ Gatsby's `build` and `develop` steps run as a Node.js application which you can
...
+- [VS Code debugger (Manual-Config)](<#vs-code-debugger-manual-config>)
```
Comment (sidharthachatterjee) [inline docs/docs/debugging-the-build-process.md]:
```suggestion
- [VS Code debugger (Manual-Config)](#vs-code-debugger-manual-config)
```

**Comment by sidharthachatterjee** [comment]
@pieh fixed the linting issues!

**Comment by muescha** [comment]
@Jezfx svgo still blocking the commits :(

**Comment by pieh** [comment]
> @Jezfx svgo still blocking the commits :(

We still merge PRs as there is nothing we can do about `svgo` ;(

**Pull request merged**